integration-tests/test_ros.py

```diff
@@ -0,0 +1,186 @@
+"""
+:component: insights-client-ros
+:requirement: RHSS-291297
+:polarion-project-id: RHELSS
+:polarion-include-skipped: false
+:polarion-lookup-method: id
+:poolteam: rhel-sst-csi-client-tools
+:caseautomation: Automated
+:upstream: Yes
+"""
+
+import pytest
+import subprocess
+import conftest
+from pathlib import Path
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+
+CONFIG_FILE = "/etc/insights-client/insights-client.conf"
+PACKAGE = "insights-client-ros"
+SERVICE = "pmlogger"
+
+
+@pytest.mark.tier1
+def test_ros_install():
+    """
+    :id: 3fc19957-be97-429d-817a-610e6c69dd9d
+    :title: Verify insights-client-ros can be installed
+    :description:
+        Ensure that the insights-client-ros can be successfully installed
+    :tags: Tier 1
+    :steps:
+        1. Try to install insights-client-ros
+        2. Check that ros_collect is set to true in insights-client.conf
+    :expectedresults:
+        1. Insights-client-ros is installed.
```

Comment:
IMO lowercase better here
```suggestion
        1. insights-client-ros is installed.
```

Reply:
It is better when the steps and expected results start with a capital letter so I have changed it a bit so that the name can be lowecased.